### PR TITLE
Bugfix/bgid push sigchld race

### PR DESCRIPTION
**Pull request by asac**

### CODE REVIEW

File: atom.mk [hunk 1]
```diff
@@ -87,6 +87,7 @@ LOCAL_SRC_FILES := debug.c \
 			utils/math.c \
 			utils/timer.c \
 			utils/fs.c \
+			utils/pvsignals.c \
```
Comment (anibalportero) [inline atom.mk:90]: this is duplicated

File: logserver/logserver.c [hunk 2]
```diff
@@ -825,6 +826,7 @@ static pid_t logserver_start_service(const char *running_revision)
 {
 	sigset_t oldmask;
 	logserver.cmd_pid = getpid();
+	sigset_t oldmask;
```
Comment (anibalportero) [inline logserver/logserver.c:829]: this one too

**Review by anibalportero — COMMENTED** [review]
I don't understand this change. Is it only adding the logs in pv_lxc?

**Comment by asac** [comment]
I made a new PR with name better matchign what this does (add logs): https://github.com/pantavisor/pantavisor/pull/385

**Pull request closed without merging**